app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -155,18 +131,41 @@ private void updateDroidCoordinates() {
         }
     }
 
+    private void sleep() {
+        try {
+            Thread.sleep(GameConstants.SLEEP_TIME);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void drawScene() {
+        if (surfaceHolder.getSurface().isValid()) {
+            Canvas canvas = getHolder().lockCanvas();
+            // Cleaning previous canvas.
```

Comment:
I would either remove all these comments or just create a general one.

It seems it is a bit redundant to have all these comments as we can infer from the name of the method what is going to happen.

Reply:
Fixed